### DISCUSSION TITLE
How are you running Trilium?

### DISCUSSION THREAD

**Discussion by meichthys** [TriliumNext/Trilium#4112] [Polls]
I'm curious to see how everyone is running Trilium since there are [a variety of methods](https://github.com/zadam/trilium/wiki/Server-installation).

**Comment by jwhonce** [comment]
flatpak on laptop, podman/quadlet as sync server (using docker hub container)

**Comment by biffbaxter** [comment]
Late to the party on this, but I am using the packaged server. Simple, quick, and I already have an NGINX hardware box running separately in my home lab so the certs and config for that are already sorted, so its low maintenance to deploy. I used to run Docker on everything, but not as much as I used to.
